app/src/main/java/io/pslab/activity/OscilloscopeActivity.java

```diff
@@ -1011,11 +1013,40 @@ protected Void doInBackground(String... channels) {
                     }
                     short[] buffer = audioJack.read();
                     yDataString.add(new String[buffer.length]);
-                    for (int i = 0; i < buffer.length; i++) {
+
+                    int n = buffer.length;
+                    Complex[] fftOut = null;
+                    if (isFourierTransformSelected) {
+                        Complex[] yComplex = new Complex[n];
+                        for (int j = 0; j < n; j++) {
+                            float audioValue = (float) map(buffer[j], -32768, 32767, -3, 3);
+                            Log.d("mic", audioValue + "");
```

Comment:
Let's remove this

Reply:
done. sorry i forgot to remove debugging statement